tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test

```diff
@@ -6,14 +6,14 @@ def docstring_missing_description(dummy):
 
 
 class docstring_class_testcase(dummy):
-    """
+    r"""
     Example docstring class.
     :param dummy:dummy description
     :return:nothing
     """
 
     def docstring_memberfunction(self, dummy):
-        """
+        r"""
```

Comment:
could you rather add new functions with new docstrings? Maybe inside a new test file. Then we keep the "old" tests still intact.